rpcs3/Emu/RSX/RSXFIFO.cpp

```diff
@@ -163,8 +164,8 @@ namespace rsx
 			}
 
 			// Validate the args ptr if the command attempts to read from it
-			m_args_ptr = RSXIOMem.RealAddr(m_internal_get + 4);
-			if (UNLIKELY(!m_args_ptr))
+			m_args_ptr = m_iotable->get_ioaddr(m_internal_get + 4);
+			if (!LIKELY(m_args_ptr + 1))
```

Comment:
is this correct ? LIKELY vs UNLIKELY also the +1

Reply:
it was correct, the boolean value is negated here so LIKELY becomes UNLIKELY. this is a check for != -1 so when adding 1 it woukd only becomes zero for it.

Reply:
the m